numpy/ma/__init__.pyi

```diff
@@ -218,6 +218,7 @@ from numpy.ma.extras import (
     masked_all_like as masked_all_like,
     median as median,
     mr_ as mr_,
+    ndenumerate as ndenumerate,
```

Comment:
FYI, you'll also have to add `ndenumerate` to the [extras.pyi](https://github.com/numpy/numpy/blob/c92864091e5928d92bc109d1505febe35f3909f1/numpy/ma/extras.pyi) stub file for this to work:
``` diff
+ from numpy.typing import ArrayLike, _Shape
+ from typing import Generator, Tuple
+
+ def ndenumerate(a: ArrayLike) -> Generator[Tuple[_Shape, Any], None, None]: ...
```